drivers/unix/net_socket_posix.cpp

```diff
@@ -335,7 +335,7 @@ Error NetSocketPosix::open(Type p_sock_type, IP::Type &ip_type) {
 
 	if (protocol == IPPROTO_UDP && ip_type != IP::TYPE_IPV6) {
 		// Enable broadcasting for UDP sockets if it's not IPv6 only (IPv6 has no broadcast option).
-		set_broadcasting_enabled(true);
+		set_broadcasting_enabled(false);
```

Comment:
The comment is no longer valid. I guess the `if` condition isn't either, it could be a plain `set_broadcasting_enabled(false);` by default (if it's not already false).